src/main.py

```diff
@@ -50,6 +73,10 @@ async def socket_listener(self, websocket: WebSocketServerProtocol):
         client_type = message_dict["client"]
         if client_type == "ui":
             await self._ui.handle_socket(websocket)
+        elif client_type == "screen":
+            component = Screen()
+            self._add_component(component)
```

Comment:
If we were to connect two screens, this would mean two components. But Screen already has a reference to each client, so is this necessary or could one Screen be created in main() like we do with UI?

If we do keep it here, we should remove it when the websocket disconnects.